Makefile

```diff
@@ -34,9 +34,9 @@ clean:  ## Clean build artifacts
 
 .PHONY: archive
 archive: clean  ## Create the archive for AWS lambda
-	docker build -t bucket-antivirus-function:latest .
+	docker build --platform linux/amd64 -t bucket-antivirus-function:latest .
```

Comment:
Does this impact non-M1 chip MacBooks?

Reply:
Just saw the comment, all good.

Reply:
should not. Assumably the default for non M1s is linux/amd64, so it's overriding the default with the same value